syntax = "proto3";

package syft.grid.duet;

import "proto/core/common/common_object.proto";

message RequestMessage {
    syft.core.common.UID request_id = 1;
    syft.core.common.UID object_id = 2;
    string request_name = 3;
    string request_description = 4;
}